actor/kvcounter/README.md

```diff
@@ -0,0 +1,19 @@
+# KeyValue Counter
+
+This actor accepts http GET requests, and
```

Comment:
Super Nit: The line wrapping in this file is real odd. I generally prefer 100 chars, but even the standard 80 chars would be better